platformio.ini

```diff
@@ -51,9 +52,9 @@ lib_deps        = ${common.lib_deps}
                   ${m5stick-c-plus.lib_deps}
 
 [env]
-platform        = espressif32
+platform        = espressif32@3.2.0
```

Comment:
Your PR description states that you aim to go  back to v3.**5**.0 of the platform. You seem to pin down the platform version to v3.**2**.0 instead?